### ISSUE TITLE
opened install.bat

### ISSUE THREAD

**Issue by pikachu-bot-info** [lnxcz/nitro-generator#3]
its not working can u tell me what i need to do and kindly put a good guide and let me know what i want to do
![isse#1](https://user-images.githubusercontent.com/71018937/94263108-2f0cf180-ff52-11ea-9dad-4915713d95a7.PNG)

**Comment by lnxcz** [comment]
Hi, ypu need to install newest python version at https://www.python.org/
Then it will work.

**Comment by pikachu-bot-info** [comment]
ohhh ty is it necessary to install it like my pc is not much updated to install it

**Comment by erick-dsnk** [comment]
> ohhh ty is it necessary to install it like my pc is not much updated to install it

edit install.bat and instead of `py -3.8` put `python`. as long as you have python 3.x installed this should fix it.

**Comment by lnxcz** [comment]
I used 3.8 beacuse i code in 3.8 and in different version may not work properly.

**Comment by erick-dsnk** [comment]
yeah but i looked through the code and as far as i can tell there's no 3.8 exclusive features, so even a somewhat recent version will do. even 3.4 will probably work

**Comment by CTRL-ALT-EXIST** [comment]
@lnxcz Tested in 3.7
Works, but after some codes, it randomly crashes. Unsure why, will document.

**Comment by CTRL-ALT-EXIST** [comment]
@pikachu-bot-info 
If this issue is solved please close it
